adapters/smaato/smaatotest/exemplary/simple-banner.json

```diff
@@ -118,7 +120,9 @@
             "coppa": 1,
             "ext": {
               "gdpr": 1,
-              "us_privacy": "uspConsentString"
+              "us_privacy": "uspConsentString",
+              "gpp": "gppString",
+              "gpp_sid": [7]
```

Comment:
Could we also add `gpp` and `gpp_sid` json for other media types as well like video and native?

Reply:
We could, but why?

Reply:
To cover the test case for other media types.

Reply:
alright, pushed

Reply:
thanks @Enigo . LGTM now!